core-bundle/contao/dca/tl_templates.php

```diff
@@ -218,6 +218,7 @@ public function addBreadcrumb()
 		$GLOBALS['TL_DCA']['tl_templates']['list']['sorting']['root'] = array($strNode);
 
 		// Insert breadcrumb menu
+		$GLOBALS['TL_DCA']['tl_templates']['list']['sorting']['breadcrumb'] ??= '';
 		$GLOBALS['TL_DCA']['tl_templates']['list']['sorting']['breadcrumb'] .= '
```

Comment:
Is this the correct fix, or should the next line just be `=` instead of `.=`?

Reply:
The latter, I think. Theoretically, there could be a callback that has already added something to `$GLOBALS['TL_DCA']['tl_templates']['list']['sorting']['breadcrumb']` but I don‘t think that is a supported use-case, is it?

Reply:
Done in ad05006a10846545d9eb2ca3d8c1ba3584c133ce